src/reducers/reducers.js

```diff
@@ -15,17 +15,18 @@ function selectedClassroom(state = {}, action ){
   }
 }
 
-function classrooms(state = {
+function allClassrooms(state = {
   isFetching: false,
-  didInvalidate: false,
   items: []
 }, action) {
   switch (action.type) {
     case types.REQUEST_CLASSROOMS:
+      console.log('REDUCERS: REQUEST_CLASSROOMS')
       return Object.assign({}, state, {
         isFetching: true
       })
     case types.RECEIVE_CLASSROOMS:
+      console.log('REDUCERS: RECEIVE_CLASSROOMS')
       return Object.assign({}, state, {
         isFetching: false,
         items: action.classrooms
```

Comment:
Not sure about this one @rogerhutchings . The context of `items` should be clear because it is within `allClassrooms`. So I don't think `items` is ambiguous. This is what I mean:
![screen shot 2016-02-11 at 10 17 33](https://cloud.githubusercontent.com/assets/8989606/12974069/04dcce62-d0a9-11e5-98fc-3334186ae08c.png)